lightning_client.go

```diff
@@ -2192,6 +2195,32 @@ func (s *lightningClient) GetChanInfo(ctx context.Context, channelId uint64) (
 	}, nil
 }
 
+// ListPeers gets a list of pubkeys of the peers we are currently connected to.
+func (s *lightningClient) ListPeers(ctx context.Context) ([]route.Vertex,
```

Comment:
Although we don't currently need all the data returned from the ListPeers RPC we ultimately might. It's up to you but maybe best to add everything now?

Reply:
yeah, likely saves somebody some future pain!